sporkfish/lichess_bot/lichess_bot_berserk.py

```diff
@@ -144,13 +145,39 @@ def set_pos_and_play_move(
                 num_moves = len(state["moves"].split())
                 prev_moves = state["moves"]
                 set_pos_and_play_move(num_moves, color, prev_moves, game_id, state)
+            elif state["type"] == "opponentGone" and state["gone"]:
+                time.sleep(8)
```

Comment:
Have you tested this works for all time controls? IIRC we need at least 15 seconds for rapid games until you can claim a victory. Anyway, lets not hardcode 8. We can directly get it from the API. https://lichess-org.github.io/berserk/api.html#berserk.clients.Board:

> Generally, this should only be called once the opponentGone event is received in the board game state stream and the claimWinInSeconds time has elapsed.

Please use claimWinInSeconds instead of 8 then.

Reply:
Btw, what happens if in-between time.sleep(claimWinInSeconds) and you claiming victory, the opponent returns?

Reply:
Also, worth adding a TODO comment: what happens if claimWininSeconds > time we have left (unlikely, but possible)?